demos/video-filtering.c

```diff
@@ -715,9 +713,9 @@ static void api1_example(void)
 // API #2: Pretend we have some fancy pool of images.
 #define POOLSIZE (PARALLELISM + 1)
 
-static struct api2_buf buffers[POOLSIZE] = {0};
-static struct image images[POOLSIZE] = {0};
-static int refcount[POOLSIZE] = {0};
+static struct api2_buf buffers[POOLSIZE];
+static struct image images[POOLSIZE];
+static int refcount[POOLSIZE];
```

Comment:
What compiler warns about this? Even if the zero-initializer can be dropped for static variables, it still serves to illustrate that fact to the reader, IMHO. (Clarity over conciseness)

Reply:
The compiler on Ubuntu 16.04 throw warning of missing initializer. It's the compiler which should be blamed, but if we can silent these false positives, we can focus on real problem if there is.
cf. https://stackoverflow.com/questions/1538943/why-is-the-compiler-throwing-this-warning-missing-initializer-isnt-the-stru

On the other hand, it cannot reproduced on Debian sid, and Ubuntu 16.04 is kind of outdated, so I can drop this one.

Reply:
For working around "bad" warnings I'd personally prefer just adding `-Wno-*` when building it, like we do for `-Wno-missing-braces` on clang. But I don't really have a strong opinion here. Maybe it's clear to all C programmers that static variables are 0-initialized (rather than undefined)?